tests/integration/podman/Containerfile_archlinux

```diff
@@ -5,3 +5,11 @@ RUN pacman -Syu --noconfirm git
 COPY ./clone_and_install_kw.sh .
 
 RUN bash ./clone_and_install_kw.sh
+
+COPY ./scripts /scripts
+
+COPY ./move_test_scripts_to_bin.sh /tmp/move_test_scripts_to_bin.sh
```

Comment:
Are you sure you want to keep this script in the `/tmp` folder from the container? From the script name, it sounds like you want to put it in /usr/bin or similar.